docs/src/assets/scss/tokens/themes.scss

```diff
@@ -132,6 +144,10 @@ html[data-theme="light"] {
     --footer-background-color: var(--color-neutral-25);
     --outline-color: var(--color-brand);
     --img-background-color: #fff;
```

Comment:
[Non-Actionable] Existing code: this `--img-background-color` is only used in `div.img-container`, which is used in `contribute/architecture` and `extend/code-path-analysis` pages. In light mode the images on those pages have a white background anyway. So 👍 there's no visual difference from manually adding this to the default `:root` styles.

Just noting as it's the only exception to the styles matching up. 🙂